elasticsearch/config/load_settings.sh

```diff
@@ -15,8 +15,13 @@ else
   wazuh_url="${WAZUH_API_URL}"
 fi
 
+if [ "x${ELASTICSEARCH_USERNAME}" = "x"]; then
+  auth=""
+else
+  auth="--user ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD}"
+fi
 
-until curl -XGET $el_url; do
+until curl ${auth} -XGET $el_url; do
```

Comment:
Better to follow the same style as the rest of the document for specifying the auth credentials: 
`curl -XGET $el_url  ${auth} `